src/sph_types.h

```diff
@@ -1057,6 +1057,13 @@ typedef long long sph_s64;
 
 #define SPH_DETECT_LITTLE_ENDIAN     1
 
+/*
+ * LoongArch, little-endian.
```

Comment:
Sorry, I missed this comment. All LoongArch systems are little-endian per the manual, so this line should simply read:

```suggestion
 * LoongArch.
```

Looking at the context around the "MIPS, little-endian" block you can find an adjacent block for "MIPS, big-endian", so LoongArch doesn't need the "little-endian" qualifier.